File: php-sdk/examples/trending-news-sources.php
<?php

/*

	File: trending-news-sources.php
	Endpoint: public/trending-news-sources
	Repo: https://github.com/executium/trending-historical-cryptocurrency-news
	License: https://github.com/executium/trending-historical-cryptocurrency-news/LICENSE

*/

include '../src/Executium.trendingnews.Class.php';

$obj = new ExecutiumTrendingNews();

# Parameters required
$payload=array();

# Send Request
$result = $obj->trending_news_sources($payload);

# Result
echo '<pre>';
	echo "\n<strong>Returned</strong>\n\n";
	print_r($result);
echo '</pre>';

?>

File: python-sdk/examples/trending-news-sources.py
'''

	Endpoint: public/trending-news-sources
	Name: Trending News Sources

'''
import sys
import json

sys.path.insert(1, '../src')

# Include config and class
from config import key, secret
from Class import ApiAccess

# Declare your key and secret
obj = ApiAccess(key, secret)

# Declare endpoint
endpoint = 'public/trending-news-sources'

# No parameters required
payload = {}

result = obj.send(endpoint, payload, dict())

print(json.dumps(result['returned'], indent=4))